conf/routes

```diff
@@ -168,14 +168,15 @@ POST /v3/recordExportStatuses      @org.sagebionetworks.bridge.play.controllers.
 
 # Studies
 GET    /v3/studies                  @org.sagebionetworks.bridge.play.controllers.StudyController.getAllStudies(format: String ?= null, summary: String ?= null)
-POST   /v3/studies                  @org.sagebionetworks.bridge.play.controllers.StudyController.createStudy 
+POST   /v3/studies                  @org.sagebionetworks.bridge.play.controllers.StudyController.createStudy
+POST   /v3/studies/createStudyAndUser   @org.sagebionetworks.bridge.play.controllers.StudyController.createStudyAndUser
```

Comment:
I dislike the use of "create" in a POST method which is always already a create. /v3/studies/init or something like that would be preferable (still not great but better).